sass/index.scss

```diff
@@ -59,1298 +143,506 @@ $white: #FFFFFF;
     }
 }
 
-// General Styles
-* {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    font-family: 'Work Sans', sans-serif;
-    line-height: 125%;
+// Pre-Reg Layout
+#pre-reg {
 
-    // Transition all elements on the screen
-    // (except FAQ panel, which has conflicting transitions)
-    // (and except certian nav stuff, which we don't want)
-    &:not(.panel):not(.show-nav):not(.left):not(.right) {
-        transition-duration: 1s;
-        transition-property: margin, padding, width, font-size;
-    }
-}
+   display: flex;
+   flex-direction: column;
+   margin: auto;
+   padding-top: 10px;
+   padding-bottom: 10px;
 
-body {
-    background-color: $off-white;
-}
+    .mini-window {
 
-// Typography
-h1 {
-    font-size: 3rem;
-    font-weight: 600;
-    font-family: 'Sora', sans-serif;
-    margin-left: -5px;
-    color: $darker-blue;
-}
-
-h2 {
-    font-size: 2.5rem;
-    font-weight: 600;
-    padding-bottom: 1rem;
-    font-family: 'Sora', sans-serif;
-    color: $darker-blue;
-}
-
-h3 {
-    font-size: 1.75rem;
-    font-weight: 600;
-    padding-bottom: 2rem;
-    color: $darker-blue;
-}
-
-p {
-    font-size: var(--body-font-size);
-    line-height: 133%;
-    padding-bottom: 1rem;
-    color: $darker-blue;
-
-    &:last-child {
-        padding-bottom: 0;
-    }
+        max-width: 656px;
+        width: 100%;
+        max-height: 316px;
```

Comment:
For future: Generally I try not to set a width **and** a height on something. I'll set a width and let the height expand, which prevents overflow issues